### ISSUE TITLE
curl_multi_wait build failure

### ISSUE THREAD

**Issue by reedjc** [dnsdb/dnsdbq#14]
Doesn't build without curl_multi_wait support which is introduced in libcurl 7.28.0. (This failure was seen in Jenkins CI.)  Debian Wheezy has older libcurl. Maybe some other projects have older libcurl too.

```
cc  -g -W -Wall -Wextra -Wcast-qual -Wpointer-arith -Wwrite-strings -Wmissing-prototypes  -Wbad-function-cast -Wnested-externs -Wunused -Wshadow -Wmissing-noreturn -Wswitch-enum -Wconversion -Wformat-nonliteral -Werror `curl-config --cflags`  -I/usr/local/include -c dnsdbq.c
dnsdbq.c: In function ‘io_engine’:
dnsdbq.c:1444:3: error: implicit declaration of function ‘curl_multi_wait’ [-Werror=implicit-function-declaration]
dnsdbq.c:1444:3: error: nested extern declaration of ‘curl_multi_wait’ [-Werror=nested-externs]
cc1: all warnings being treated as errors
```

This debian system has installed
libcurl3:amd64 7.26.0-1+wheezy24
libcurl4-openssl-dev 7.26.0-1+wheezy24

**Comment by vixie** [comment]
i don't anticipate modifying dnsdbq to no longer rely on multiple simultaneous fetches. if that were possible, it would be the default. yes, i could fork() and initialize a new curl instance in the child, but that would be crazy. i already fork() to call POSIX sort, that's as far as we ought to go down that track. as i see it we will either have to document a need for some kind of "wheezy backports" dependency, or decertify wheezy as a compatible dnsdbq platform. what's your advice?

**Comment by reedjc** [comment]
Okay. Let's mention the minimum libcurl version in the README and change the "Debian 7" to "Debian 8" in the README. (Debian 8 has much newer libcurl.)  I am fine with it not building on older. Later we can consider autoconf to check for good-enough dependencies.

**Comment by vixie** [comment]
sounds good. can you submit the PR?

**Comment by vixie** [comment]
fixed in 2058f3a.
